dags.yaml

```diff
@@ -539,6 +539,26 @@ bqetl_internal_tooling:
   tags:
     - impact/tier_3
 
+bqetl_internal_tooling_phase_2:
```

Comment:
I'm not really sure if this is the best way to handle this or not, I'm open to other suggestions. Basically this table depends on the `fxci_derived.worker_costs_v1` table, so needs to run afterwards.

Reply:
I don't think a new DAG is needed here. The new query can be part of the existing `bqetl_internal_tooling` DAG. bigquery-etl will automatically determine the right order in which the tasks need to run just based on the SQL queries and referenced tables

Reply:
Fancy!